app/src/app.vue

```diff
@@ -99,6 +103,8 @@ useSystem();
 
 			<template #append>
 				<v-error :error="error" />
+				<br />
+				<v-button kind="danger" @click="reload()">{{ t('reload') }}</v-button>
```

Comment:
Should this actually be a `danger` button or is `normal` fine?

Reply:
Normal 👍

Reply:
Could also be "Reload Page"

Reply:
Changes were pushed via https://github.com/directus/directus/pull/22462/commits/4883f92c569efa7e91c948a70d095c057d29372c and https://github.com/directus/directus/pull/22462/commits/ae0b8c8f47c6f27e7e9ef669fec22c1ea1f3014c accordingly. Thank you both! 🚀